test/action.yml

```diff
@@ -0,0 +1,92 @@
+---
+name: Test Python
+description: |
+  Github Action to Build & Test Python
+
+  RequiredEnv:
+    GITHUB_TOKEN
+    SONAR_TOKEN
+inputs:
+  python-version:
+    description: |
+      The python-version input is optional.
+      If not supplied, the action will try to resolve the
+      version from the default `.python-version` file.
+
+      If the `.python-version` file doesn't exist Python or PyPy
+      version from the PATH will be used.
+
+      The default version of Python or PyPy in PATH varies
+      between runners and can be changed unexpectedly
+      so we recommend always setting Python version explicitly
+      using the python-version inputs.
+    required: false
+runs:
+  using: composite
+  steps:
+    - name: Find Python Packager
+      id: packager
+      shell: bash
+      run: |
+        echo "packager=pip" >> $GITHUB_OUTPUT
+        if [ -f "requirements.txt" ]; then
```

Comment:
Isn't this redundant? As in line above it's already set as pip.

Reply:
removed